docs/pages/2019_MARVEL_Psik_MaX/sections/calculations.rst

```diff
@@ -263,44 +241,42 @@ the ``parameters_dict`` on a ``Dict`` model:
 Simulate submission
 ~~~~~~~~~~~~~~~~~~~
 
-At this stage, you have recreated in memory (it’s not yet stored in the
-database) the input of the graph shown bellow, whereas the outputs will
-be created later by the daemon.
+At this stage, you have created in memory (it’s not yet stored in the
+database) the input of the graph shown below. The outputs will
+be created by the daemon later on.
 
 .. figure:: include/images/verdi_graph/si/graph-full.png
    :alt:
 
-In order to check how AiiDA creates the actual input files for the
-calculation, we can perform a *dry run* of the submission process, to
-achieve that, we should modify the metadata of our builder, first
-specifying that we want a dry run, and then telling it that we don't
-want to store the provenance of this calculation (yet):
+In order to check which input files AiiDA creates, 
+we can perform a *dry run* of the submission process.
+Let's tell the builder that we want a dry run and that
+we don't want to store the provenance of the dry run:
 
 .. code:: python
 
     builder.metadata.dry_run = True
     builder.metadata.store_provenance = False
 
-After setting the appropriate metadata, is time for us to actually run
-the *dry run*, to do that we need to reach for AiiDA's run
-functionality:
+It's time run:
 
 .. code:: python
 
     from aiida.engine import run
     run(builder)
 
 This creates a folder of the form ``submit_test/[date]-0000[x]`` in the
-current directory. Check (in your second terminal) the input file
-``aiida.in`` within this folder, comparing it with the content of the
-input data nodes you created earlier, and that the ‘pseudo’ folder
-contains the needed pseudopotentials. You can also check the submission
-script ``_aiidasubmit.sh`` (the scheduler that is installed on the
-machine is Torque, so AiiDA creates the files with the proper format for
-this scheduler). Note: you cannot correct the input file from the
-``submit_test`` folder: you have to correct the script and re-execute
-it; the files created by ``submit_test()`` are only for final
-inspection.
+current directory. In your second terminal:
+
+ * open the input file ``aiida.in`` within this folder
+ * compare it to input data nodes you created earlier
+ * verify that the `pseudo` folder contains the needed pseudopotentials
+ * have a look at the submission script ``_aiidasubmit.sh`` 
+   
+.. note:: 
+
+   The files created by ``submit_test()`` are only intended for  inspection
```

Comment:
Replace ``submit_test`` by something like "performing a dry-run"

Reply:
by the way, should we add a note on the difference between `run`  and `submit` here (because it would give the impression that `run` is for dry runs and `submit` is for actual calculations.

something like 
```
.. note::

   You may have noticed that we used ``submit`` here instead of ``run``.
   ``submit`` and ``run`` work just the same, except that ``submit`` will hand over the calculation to the daemon,
   making sure that everything can execute in the background, while ``run`` will execute all tasks in the current
   shell. Use ``run`` for testing and debugging, and ``submit`` for production work.
```